README.md

```diff
@@ -35,6 +36,9 @@ With `make sign` & `make exec`, one can also use any other wallet provider avail
 - `make cmd:keystore` to use a keystore
 - `make cmd:"private-key 0x..."` if you really want to save your private key to your shell's history...
 
+You can also append any `cast` parameter:
+- `make cmd:"ledger --mnemonic-indexes 1"` to use another account than the default one at index `0`
```

Comment:
```suggestion
- `make cmd:"ledger --mnemonic-index 1"` to use another account than the default one at index `0`
```

Reply:
The foundry book seems to indicate it is plural: https://book.getfoundry.sh/reference/cast/cast-wallet-address

Reply:
Ok after having tested w the latest forge version (forge 0.2.0 (31fcf5a 2023-05-19T00:04:19.980323193Z)), using plural for `cast wallet sign` gave me the following error:
![image](https://github.com/morpho-labs/safer/assets/61523188/7df2b35c-f586-44c3-884f-5fd09a157308)

But, for exec, only plural works
```bash
 make exec:"ledger --mnemonics 1  --mnemonic-indexes 1"
```

Reply:
I think you can document the 2 workflows (sign & exec) with the current working flow